pages/partials/questionNavSideButtonGroup.ejs

```diff
@@ -0,0 +1,17 @@
+<div class="text-center mb-4">
+  <%- include("../partials/questionNavSideButton", {
+    button: {
+      "id": "question-nav-prev",
+      "label": "Previous question",
+    },
+    question: instance_question_info.prev,
+  }); %>
+  <%- include("../partials/questionNavSideButton", {
+    button: {
+      "id": "question-nav-next",
+      "label": "Next question",
+    },
+    question: instance_question_info.next,
+    unlock_score: instance_question_info.advance_score_perc,
```

Comment:
Can we avoid renaming variables? That is, just call this `advance_score_perc` rather than `unlock_score`. Code is _much_ easier to follow when the same value gets copied into variables with the same name (or at least almost the same name).

Reply:
Done